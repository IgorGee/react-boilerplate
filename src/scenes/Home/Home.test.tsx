import React from 'react'
import { shallow } from 'enzyme'
import Home from './'

describe('<Home />', () => {
  it('renders initial UI', () => {
    const component = shallow(<Home />)
    expect(component).toMatchSnapshot()
  })
})
