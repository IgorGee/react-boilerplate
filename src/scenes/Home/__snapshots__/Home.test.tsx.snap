// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Home /> renders initial UI 1`] = `
<div>
  <styled.h1>
    Home
  </styled.h1>
</div>
`;
